src/stats.php

```diff
@@ -131,8 +131,15 @@ function getContributionGraphs(string $user): array
     }
     // extract the year from the created datetime string
     $userCreatedYear = intval(explode("-", $userCreatedDateTimeString)[0]);
+    // if override parameter is null then define starting year
+    // as the user created year; else use the provided override year
+    if (is_null($overrideMinYear)) {
+        $minimumYear = $userCreatedYear;
+    } else {
+        $minimumYear = intval($overrideMinYear);
+    }
```

Comment:
Ternary or ternary shorthand can be used here to simplify
```suggestion
    $minimumYear = $overrideMinYear ?: $userCreatedYear;
```